cluster/examples/kubernetes/ceph/admission-controller/hack/deploy.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
We will need documentation that instructs the admin how to enable an admission controller. I'm thinking we'll need a new .md file under the Documentation folder. You already have some of this in the design doc, but we need to turn it into a user-friendly doc that talks about how to configure it, rather than the implementation details.

Reply:
How about adding a doc as part of this PR? I'd suggest the header of the .md be this so that it will be between the Prereqs and Examples in the table of contents:
```
---
title: Admission Controller
weight: 2030
indent: true
---

# Admission Controller

...
```